vnext/src/Libraries/Components/Flyout/FlyoutProps.ts

```diff
@@ -24,7 +24,9 @@ export interface IFlyoutProps extends ViewProps {
   horizontalOffset?: number;
   isLightDismissEnabled?: boolean;
   isOpen?: boolean;
+  attachAsContextFlyout?: boolean;
```

Comment:
>attachAsContextFlyout [](start = 2, length = 21)

Suggestion: isContextMenu

It's shorter and I think it matches other prop names a little bit closer.

Reply:
what about isContextFlyout? Because the flyout may not necessarily be menu.

Reply:
Works for me :)